src/axom/quest/interface/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(YAML_INPUT_FILE quest_shroud.yaml)
 
 add_shroud(
     YAML_INPUT_FILE      ${YAML_INPUT_FILE}
+    YAML_OUTPUT_DIR      yaml
```

Comment:
Is that new in `shroud`? What does it do exactly?

Reply:
Shroud generates `yaml` files as part of its interface files.  This tells it where to place them.

When I ran shroud on quest, I noticed that two files were missing, and the yaml file was at the base directory.
(Note: There doesn't seem to be too much in the quest yaml file).

Reply:
Thanks for the explanation -- I wasn't aware of that parameter.